LoopKitUI/Views/DescriptiveText.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  DescriptiveText.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-02-20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+public struct DescriptiveText: View {
```

Comment:
[nit] Not sure this small styling (footnote, foreground color) is worth promoting to design language level, particularly since the foreground color will depend on context.

Reply:
At least with the Dexcom UI there are a number of places (currently 5) that this view is used. Being able to change their style consistently seems less prone to error and more efficient to me.